content/lua/concepts/iterators/iterators.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: 'Iterators' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Iterators are used to loop through a collection of data. They are also used to repeat processes a preset or infinite amount of times.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Computer Science'
+  - 'Lua'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Iterators'
+  - 'Conditionals'
+  - 'Tables'
+  - 'Arrays'
+  - 'Logical'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'lua/iterators'
+---
+
+# **Iterators**
+
+**Iterators** are used to loop through a collection of data. They are also used to repeat processes a preset or infinite amount of times.
+
+## **Iteration Types**
+
+In Lua, there are multiple ways in which we can **iterate** through and/or repeat code with. Some iteration methods are intended for iterating through **Arrays**, while others can be used to
+**repeat** a command until/while a condition is met. Below are some examples of conditional iterations.
+
+```lua
+-- while CONDITION do
+-- # This iteration will repeatedly execute it's provided code until the CONDITION becomes false.
+x = 2
+
+while x == 2 do
+    print("x is still equal to 2!")
+end
+
+-- repeat TASK until CONDITON
+-- # This iteration will repeat its provided task until the required CONDITION becomes true
+i = 1
+
+repeat
+    i = i + 1
+    print(i) -- 1, 2, 3, 4, ..., 20
+until
+i == 20 -- Stops iteration when 'i' becomes equivalent to 20
+```
+
+As mentioned, we can also use iterations to loop *through* an Array. To accomplish this, we must use the global keyword **pairs**. **pairs** allows us to loop through an Array in order.
```

Comment:
```suggestion
As mentioned, iterations can be used to loop *through* an array. For this, the global keyword **pairs** is used loop through an array in order.
```

Array is not a proper noun, it doesn't need to be capitalized. Also changed the phrasing to not use "we"